### PR TITLE
Refactor many object to remove code from DMM to DM code.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
create many subtype to clean dmm code

# Explain why it's good for the game
because making sub-type is good for now and the future...

# Testing Photographs and Procedure
pass the tests server launched fine..
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
refactor: create many sub-type for item to clean the code.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
going to go further and do it for a lot more items except those on almayer because it's a pain

**Comment by Huffie56** [comment]
> image 🙇‍♂️ god be mercifull of the reviewers

image

harry seem fine with it.

**Comment by Huffie56** [comment]
i guess  you could check the map change bot and the dm filed i modified to check if i screw up... normally nothing should be changed only code...

**Review by harryob — CHANGES_REQUESTED** [review]
corsat got changed

**Comment by Huffie56** [comment]
> corsat got changed

you mean science annex with this PR ?https://github.com/cmss13-devs/cmss13/pull/3288
because i don't see any PR changing corsat...
or do you mean i shouldn't change corsat?

**Comment by Huffie56** [comment]
so the issue was that i forgot some stuff are attach to weedable so i fix this by updating their path and properly update this
image
going to check the rest of the maps to see if i miss something...

**Comment by Huffie56** [comment]
> image 🙇‍♂️ god be mercifull of the reviewers

image

File: code/game/turfs/strata.dm [hunk 1]
```diff
@@ -5,6 +5,13 @@
 /turf/open/gm/grass/weedable/is_weedable()
 	return FULLY_WEEDABLE
 
+//just in case
+
+/turf/open/gm/grass/grass1/weedable/ //inherit from general turfs
```
Comment (Zonespace27) [inline code/game/turfs/strata.dm:10]:
```suggestion
/turf/open/gm/grass/grass1/weedable //inherit from general turfs
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
all the `/dir_NUMBER` subtypes should be `/north`, `/south`, etc.

File: code/game/turfs/strata.dm (hunk 1 above)
Comment (Huffie56) [reply code/game/turfs/strata.dm:10]: this is probably not on only one instance.... will fix that....

**Comment by Huffie56** [comment]
> all the `/dir_NUMBER` subtypes should be `/north`, `/south`, etc.


yes that would be great.... but the issue is that some of those item using are corners so would probably mean more using 
NE NW SE SW?
also their is some grass tiles using random direction but i think that's for another PR....

**Comment by Huffie56** [comment]
> all the `/dir_NUMBER` subtypes should be `/north`, `/south`, etc.

done.

**Comment by Huffie56** [comment]
massive new addition incoming

**Comment by Huffie56** [comment]
yep i mess it up it seem...

**Comment by Nanu308** [comment]
You need to start atomizing your changes to per map in separate PR's, this will take forever to go through and risks conflicts

**Review by harryob — APPROVED** [review]
another awesome pr tbh here's hoping nothing blows up!

**Pull request merged**